test/scenarios.py

```diff
@@ -7874,4 +7872,87 @@
             ),
         },
     ),
+    Scenario(
+        "scenario39-1",
+        "Ensure Receive event canceling field is not set for fresh transfer",
+        {
+            "ren": ActionList(
+                [
+                    action.SleepMs(400),
+                    action.Start("172.20.0.5"),
+                    action.NewTransfer("172.20.0.15", ["/tmp/testfile-small"]),
+                    action.Wait(
+                        event.Queued(
+                            0,
+                            {
```

Comment:
I guess the tests here are no longer valid, and by looks of it `scenario4-12` is doing this test, albeit it's completely broken :D

Reply:
Oh, yeah. Good point. Let me copy paste 39-2 into 4-12 and remove 31-1